js/app.js

```diff
@@ -0,0 +1,259 @@
+console.log('The Chamber of Secrets has been opented. Enemies of the Heir beware.')
+/* 1. A light switch that can be either on or off.
+boolean: light on == true
+
+2. A user's email address.
+string: "me@me.com"
+
+3. A spaceship with a hull, laser blasters, tractor beam, and warp drive.
+array: let spaceship = ['hull', 'laser blasters', 'tractor beam', 'warp drive']
+
+4. A list of student names from our class.
+array: let studentNames = ['Kait', 'Ryne', 'Travis', 'Sergio', 'Sierra', 'Jackson']
+
+5. A list of student names from our class, each with a location.
+object: let stuNameAndLoc = {
+	Stu1: {
+		name: 'Kait',
+		location: 'St. Louis'
+	}
+
+	Stu2: {
+		name: 'Michael',
+		location: 'Chicago'
+	}
+
+	Stu3: {
+		name: 'Travis',
+		location: 'Portland'
+	}
+
+	Stu4 {
+		name: 'Sergio',
+		location:'Austin'
+	}
+}
```

Comment:
movies are not tv shows but this isn't a film + tv class, so ok this time
**6/6** for this section